dbms/src/Flash/tests/gtest_compute_server.cpp

```diff
@@ -22,7 +22,9 @@ namespace DB
 namespace FailPoints
 {
 extern const char exception_before_mpp_register_non_root_mpp_task[];
+extern const char exception_before_mpp_make_non_root_mpp_task_public[];
```

Comment:
Looks like a ut can be added, send two mppquery with the same query_id

Reply:
Not very easy since the task id is always start from 1, so if just send to mpp query with the same query id, it could meet task registered error.

Reply:
ok